aspnetcore/release-notes/aspnetcore-8.0.md

```diff
@@ -121,6 +121,41 @@ builder.On<string, string>("ReceiveMessage", (user, message) => ...
 await builder.StartAsync();
 ```
 
+### SignalR seamless reconnect
+
+SignalR seamless reconnect reduces the perceived downtime of clients that have a temporary disconnect in their network connection, such as when switching network connections or a short temporary loss in access.
+
+Seamless reconnect achieves this by:
+
+* Temporarily buffering data on the server and client.
+* Acknowledging messages received (ACK-ing) by both the server and client.
+* Recognizing when a connection is returning and replaying messages that may have been sent while the connection was down.
+
+To opt-in to seamless reconnect, update your .NET client code to enable the `UseAcks` option:
+
+```csharp
+var hubConnection = new HubConnectionBuilder()
+    .WithUrl("<hub url>",
+             options =>
+             {
+                options.UseAcks = true;
+             })
+    .Build();
+```
+
+Seamless reconnect was introduced in .NET 8 [.NET 8 Preview 5](https://devblogs.microsoft.com/dotnet/asp-net-core-updates-in-dotnet-8-preview-5/#signalr), is still being designed, and has the following limitations:
+
+* There isn’t any configuration yet.
+* Support is limited to .NET clients using WebSockets.
+
+Likely features to be added in future previews of SignalR seamless reconnect:
```

Comment:
You could consider leaving these details out, as the link to https://github.com/dotnet/aspnetcore/issues/46691 provides this kind of information.

Reply:
Good point. Looking at it again, I think I still want to leave the limitations since they readily come to mind as questions as soon as you want to try the code.

I could remove the "Likely features to be added" and rely on the link for that.   @BrennanConroy in case he has a preference.

Reply:
Remove, it's already hard enough keeping docs up to date, let's not add more potential for stale data.